index.html

```diff
@@ -322,6 +322,36 @@ <h3 class="skills_dropdown_title dropdown_child_class">
                   </li>
                </ul>
            </section>
+
+           <section class="contact" id="contact">
+            <div class="contact__container">
+              <h2 class="contact_header">Contact me</h2>
+              <p class="contact_description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+              </p>
+              <form action="#" method="post">
+                <ul>
+                  <li>
+                    <label for="name">Name:</label>
+                    <input type="text" id="name" name="firstname" placeholder="Enter your name here" maxlength="30" required>
```

Comment:
- [ ] Please make sure to use the correct input type, in this case, it'd be "email" instead of "text" you can read more about input types here: https://www.w3schools.com/html/html_form_input_types.asp

![image](https://user-images.githubusercontent.com/73140461/148842013-6ed346b0-b7ad-4173-a087-6724f2c57b7a.png)

Reply:
I made a mistake here, this comment refers to line 342 instead of line 338, sorry about that.